docs/general/teams/devquest-team.md

```diff
@@ -0,0 +1,307 @@
+
+# DevQuest Team
+
+The DevQuest team is responsible for processing DevQuest requests from developers, determining what games, sets, and other actions are appropriate for the various DevQuests, voting on cases related to this, and for planning new DevQuests and the rules for all DevQuests.
+
+[[toc]]
+
+
+## DevQuest Team Responsibilities
+
+Everything related to DevQuests is the responsibility of the team:
+
+- Answering questions by developers
+- Voting on decisions related to DevQuests, which may include set eligibility, new quests, adding games to lists, etc.
+- Adding Wish This Set games to the DQ16 games list
+- Verifying games, sets, and other work for credit towards a DevQuest when pinged by a developer asking for credit.
+
+## Voting Policy
+
+For most decisions that need group input, create a new thread in the team channel on discord. It is best to have a discussion first except for simple decisions where not much information is needed. Ping @DevQuest to ensure the team is added to the thread.
+
+After any needed discussion, voting should be done using a 3-day duration discord poll.  If the poll reaches a 60% majority of the entire team before the end, then the majority action may be taken immediately.  For close votes (1 vote difference), it is recommended that further discussion take place and another follow-up vote be taken.
+
+## Adding Games to Lists
+
+Some DevQuests keep a list of eligible games and will have games added by the DevQuest team:
+
+- DevQuest 002: Retro Renovator
+- DevQuest 013: complete.me
+- DevQuest 016: Wish This Set
+- DevQuest 022: This Belonged in a Museum!
+
+### How to Choose DQ2, DQ13, DQ22 ?
+
+1. If a set requires a "repair" such as adding or fixing protection (cheat, save, password, etc.), fixing an exploit that renders the set trivial, or fixing widespread ticket-causing issues, this set should be assigned to DevQuest 002 - Retro Renovator
+2. If the set is missing some content that would render the set "incomplete" (missing progression, missing notable side content, etc.), this set should be assigned to DevQuest 013 - complete.me.
+3. If a set needs code note clean up, rich presence improvements, and logic updates, but is otherwise stable and not missing notable content, that set should be assigned to DevQuest 022 if the amount of effort needed is significant to warrant full credit.
+   - Sets that need repairs should not be assigned to DevQuest 022; a user may only use such sets for this quest if they have fully completed DevQuest 002 and then only if the non-repair-related work matches the DevQuest 022 dust-off goal.
+
+::: tip DUAL CREDIT
+If a set requires repairs that are appropriate for DQ2 and also has notable missing content that is appropriate for DQ13, it may appear in both lists. A developer could then do both actions and receive credit towards both. This is not considered "double-dipping" since the actions involved at distinct (repair as one action, completion as a separate action).
```

Comment:
```suggestion
If a set requires repairs that are appropriate for DQ2 and also has notable missing content that is appropriate for DQ13, it may appear in both lists. A developer could then do both actions and receive credit towards both. This is not considered "double-dipping" since the actions involved are distinct (repair as one action, completion as a separate action).
```